<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>prod_coming</class>
 <widget class="QWidget" name="prod_coming">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>584</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>88</y>
     <width>83</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Накладна №</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>88</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>48</y>
     <width>118</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Вид переміщення</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>342</x>
     <y>88</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="specialValueText">
    <string/>
   </property>
   <property name="displayFormat">
    <string>yyyy.MM.dd</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>372</x>
     <y>530</y>
     <width>125</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>ОК</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>120</y>
     <width>646</width>
     <height>381</height>
    </rect>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>10</y>
      <width>620</width>
      <height>321</height>
     </rect>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>338</y>
      <width>125</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Додати</string>
    </property>
    <property name="icon">
     <iconset resource="ico.qrc">
      <normaloff>:/ico/Button Add.ico</normaloff>:/ico/Button Add.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>314</x>
      <y>338</y>
      <width>125</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Видалити</string>
    </property>
    <property name="icon">
     <iconset resource="ico.qrc">
      <normaloff>:/ico/Button Delete.ico</normaloff>:/ico/Button Delete.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>48</y>
     <width>311</width>
     <height>26</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>8</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Прихід на склад</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>30</y>
     <width>651</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>292</x>
     <y>88</y>
     <width>32</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>512</x>
     <y>530</y>
     <width>125</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Скасувати</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>0</y>
     <width>651</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>500</y>
     <width>651</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="ico.qrc"/>
 </resources>
 <connections/>
</ui>
